doc/source/performance.rst

```diff
@@ -0,0 +1,100 @@
+.. _performance_considerations:
+
+.. currentmodule:: pandas
+
+.. ipython:: python
+   :suppress:
+
+   import pandas as pd
+   import numpy as np
+   np.random.seed(123456)
+
+******************
+Performance Considerations
+******************
```

Comment:
I think length of the `*` has to match exactly the length of the header, otherwise sphinx complains.